scripts/benchmark_stats.py

```diff
@@ -89,6 +102,16 @@ def _process_file_data(self):
                         )
                         self._requests[row["Type"]]["total"] += request_count
 
+    @property
+    def pdf_percentile(self) -> Any:
```

Comment:
Can this be made more accurate so we don't return `Any`, for example you can have default return here as well that returns `None` so we have better type?

Reply:
Updated 👍

Reply:
Did you push the changes you've showed me? I can still see the `Any` type hint in the file.

Reply:
Pushed this up now 👍